lib/archethic_web/controllers/api/schema/nft_ledger.ex

```diff
@@ -26,8 +27,10 @@ defmodule ArchethicWeb.API.Schema.NFTLedger do
 
   defp changeset_transfers(changeset, params) do
     changeset
-    |> cast(params, [:to, :amount, :nft])
-    |> validate_required([:to, :amount, :nft])
+    |> cast(params, [:to, :amount, :nft, :nft_id])
+    |> validate_required([:to, :amount, :nft, :nft_id])
     |> validate_number(:amount, greater_than: 0)
+    |> validate_number(:nft_id, greater_than_or_equal_to: 0)
+    |> validate_number(:nft_id, less_than_or_equal_to: 255)
```

Comment:
validate inclusion would be better option to check value in range 
validate_inclusion(changeset, :nft_id, 0..255)
https://hexdocs.pm/ecto/Ecto.Changeset.html#validate_inclusion/4